pkg/providers/hashicorp.go

```diff
@@ -67,21 +67,11 @@ func (g *hashiCorp) listReleases(repoName string) (*hashiCorpRepo, error) {
 	return &repo, nil
 }
 
-type hashiCorpFileInfo struct {
-	url, name string
-	score     int
-}
-
-func (g *hashiCorpFileInfo) String() string {
-	return g.name
-}
-
```

Comment:
I imagine these are removed because they are unused?

Reply:
Yes, correct. The `unused` linter kicked in here.